.circleci/config.yml

```diff
@@ -1,1059 +1,11 @@
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - license-scan
-            - image-scan
-          filters:
-            branches:
-              only:
-                - main
-                - /release\/v.*/
-      - github-release:
-          context: org-global
-          requires:
-            - release
-          filters:
-            branches:
-              only:
-                - main
-                - /release\/v.*/
-      - publish-docker:
-          context: org-global
-          requires:
-            - build-local
-            - pr-tools/pr-title-check
-            ## Only do this check on PRs
-            # - test-dependencies
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - license-scan
-            - image-scan
-          filters:
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*/
-            branches:
-              ignore:
-                - /.*/
-      - publish-docker-snapshot:
-          context: org-global
-          requires:
-            - build-local
-            - pr-tools/pr-title-check
-            ## Only do this check on PRs
-            # - test-dependencies
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - license-scan
-            - image-scan
-          filters:
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*\-snapshot+((\.[0-9]+)?)/
-            branches:
-              ignore:
-                - /.*/
-      - publish-npm:
-          context: org-global
-          requires:
-            - pr-tools/pr-title-check
-            ## Only do this check on PRs
-            # - test-dependencies
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - license-scan
-            - image-scan
-          filters:
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*/
-            branches:
-              ignore:
-                - /.*/
-      - publish-npm-snapshot:
-          context: org-global
-          requires:
-            - pr-tools/pr-title-check
-            ## Only do this check on PRs
-            # - test-dependencies
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - license-scan
-            - image-scan
+      - build/workflow:
```

Comment:
Like this change but need to make sure we're not missing any step. please confirm. I didn't find the license scan and another scan there on the orb, could've missed it but would like to confirm

Reply:
License and vulnerability scans is present. See example here https://app.circleci.com/pipelines/github/mojaloop/central-ledger/2809/workflows/b9859b2a-253d-483b-80b8-3e395393bd11

Reply:
Thanks!